test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test/LineMappingsSerializer.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Text;
+using Microsoft.AspNetCore.Razor.Evolution;
+
+namespace Microsoft.AspNetCore.Razor.Extensions
+{
+    public static class LineMappingsSerializer
```

Comment:
Yup, https://github.com/aspnet/Razor/issues/991. Should update that guy to reflect the fact that we no longer need a package (all codegen tests now live in Razor). We do however, need a separate infrastructure project